Formula/cyrus-sasl.rb

```diff
@@ -0,0 +1,38 @@
+class CyrusSasl < Formula
+  desc "Simple Authentication and Security Layer"
+  homepage "https://www.cyrusimap.org/sasl/"
+  url "https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.27/cyrus-sasl-2.1.27.tar.gz"
+  sha256 "26866b1549b00ffd020f188a43c258017fa1c382b3ddadd8201536f72efb05d5"
+
+  keg_only :provided_by_macos
+
+  depends_on "openssl@1.1"
+
+  def install
+    system "./configure",
+      "--disable-macos-framework",
```

Comment:
What does this flag do? Do we not want the framework when building on macOS?

Reply:
Without this flag `make install` failed when it tried to install to `/Library/Frameworks`. We could hack around that, but I don't feel the framework is needed. It installs the dylib and header files. The framework could be added back in a subsequent PR if someone really wanted it.

Reply:
Sounds good to me.